index.d.ts

```diff
@@ -44,7 +45,7 @@ declare module 'react-native-toast-message' {
 
   export default class Toast extends React.Component<ToastProps> {
     static show(options: {
-      type: string,
+      type: ToastType,
```

Comment:
there can be custom toast types as well. I'd say we leave this a `string`

Reply:
Right - I wonder if we can use a generic to infer the possible valid values from the `Toast` config dynamically. It would be nice to have some compile-time type checking on valid values here - let me see what I can do. 

Otherwise, I'll revert this bit 👍

Reply:
sure, thank you!

Reply:
I've tried a few things, and while I'm sure it can be done, I don't have the typing expertise to do it at the moment. It's also not a dealbreaker for me since at least we do get a runtime error if the type isn't found, so in practice I think it'll be manageable as-is for now.

Reply:
indeed, it's fine by me too. but thanks for looking into it anyway! 

I'm merging it now